htdocs/product/note.php

```diff
@@ -54,13 +54,13 @@
 
 if ($object->id > 0) {
 	if ($object->type == $object::TYPE_PRODUCT) {
-		restrictedArea($user, 'produit', $object->id, 'product&product', '', '');
+		restrictedArea($user, 'product', $object->id, 'product&product', '', '');
```

Comment:
inside the code of restrictedArea, we found a line that transform "product" into "produit".
So i don't think this will change something. 
Can you reproduce the bug without the change and do you experience a fix after the change ?

Reply:
yes you are write it hasn't changed anything , i changed because if the "else" was executed